src/com/mambu/apisdk/services/SearchService.java

```diff
@@ -121,21 +120,97 @@ public SearchService(MambuAPIService mambuAPIService) {
 	 *            pagination limit. If not null it must be an integer greater than zero
 	 * 
 	 * @return list of entities of the searchEntityType matching provided filter constraints
-	 * @throws MambuApiException
+	 * @throws MambuApiException in case exception occurs while fetching entities
 	 */
+	@Deprecated

```

Comment:
Should we have also a note included in javadoc, to specify that this method is now deprecated?

Reply:
yep you`re right